api/commands/run.ts

```diff
@@ -56,7 +59,9 @@ export class Runner {
   constructor(
     private readonly dbadapter: dbadapters.IDbAdapter,
     private readonly graph: dataform.IExecutionGraph,
-    partiallyExecutedRunResult: dataform.IRunResult = {}
+    partiallyExecutedRunResult: dataform.IRunResult = {},
+    runnerNotificationPeriodMillis: number = RUNNER_NOTIFICATION_PERIOD_MILLIS,
```

Comment:
these should both have `private readonly` infront of them, then you can use them below (see other comment)